gcp/src/main/java/org/apache/iceberg/gcp/GCPProperties.java

```diff
@@ -49,6 +49,12 @@ public class GCPProperties implements Serializable {
   public static final String GCS_OAUTH2_REFRESH_CREDENTIALS_ENDPOINT =
       "gcs.oauth2.refresh-credentials-endpoint";
 
+  // Impersonation properties
+  public static final String GCS_IMPERSONATE_SERVICE_ACCOUNT = "gcp.impersonate.service-account";
+  public static final String GCS_IMPERSONATE_LIFETIME_SECONDS = "gcp.impersonate.lifetime-seconds";
+  public static final String GCS_IMPERSONATE_DELEGATES = "gcp.impersonate.delegates";
+  public static final int GCS_IMPERSONATE_LIFETIME_SECONDS_DEFAULT = 3600;
+
```

Comment:
Did you mean to prefix these with `gcs`? instead of `gcp`? This looks like it overlaps with the ones defined in BigQueryProperties.  Also, it doesn't appear that we're doing anything with these, so why include them?

Reply:
Also, no scopes necessary here?